client/components/app-promo/index.jsx

```diff
@@ -69,7 +69,7 @@ export default React.createClass( {
 			return null;
 		}
 
-		var promo_link = 'https://desktop.wordpress.com/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
+		var promo_link = 'https://apps.wordpress.com/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
```

Comment:
It seems that the direct link to the desktop app is `https://apps.wordpress.com/desktop/` (and also this is the URL that `desktop.wordpress.com` redirects to).